test/extended/operators/operators.go

```diff
@@ -203,6 +211,16 @@ var _ = g.Describe("[Feature:Platform] Managed cluster should", func() {
 	})
 })
 
+func inAvailableAndFailingWhitelist(coName string) bool {
+	whitelist := map[string]struct{}{
+		"marketplace": {},
```

Comment:
If you're going to inline the map (vs. keeping it in an external variable), you might as well use a `switch`:

```go
switch coName {
case "marketplace": return true
default: return false
```

Reply:
> Failing is now Degraded

@mfojtik So are you saying that this test as written is wrong? It seems to me like the current state of ClusterOperator still has the Failing field. For example, all of the clusteroperator code in the openshift-marketplace operator still refers to it as Failing.

Or are you saying that this change is in progress? How is something like that going to get communicated to operator owners that already write that field? If it is merged, can you point me to a commit?

Reply:
Fair point. Updated in latest push.